examples/tools/ros/rosenv/workspace/app/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+cmake_minimum_required(VERSION 3.8)
+project(app)
+
+message("[app] AMENT_PREFIX_PATH env: $ENV{AMENT_PREFIX_PATH}")
+message("[app] COLCON_PREFIX_PATH env: $ENV{COLCON_PREFIX_PATH}")
+message("[app] COLCON_CURRENT_PREFIX env: $ENV{COLCON_CURRENT_PREFIX}")
+message("[app] CMAKE_PREFIX_PATH env: $ENV{CMAKE_PREFIX_PATH}")
+
+if(CMAKE_COMPILER_IS_GNUCXX OR CMAKE_CXX_COMPILER_ID MATCHES "Clang")
+  add_compile_options(-Wall -Wextra -Wpedantic)
+endif()
+
+# find dependencies
+find_package(ament_cmake REQUIRED)
+find_package(consumer REQUIRED)
```

Comment:
Having the dependency called "consumer" might be a bit confusing.
We always refer in conan to the "consumer" as the most end-consumer, in this case, the "consumer" seems this "app" executable, and not the other which seems a dependency, probably need to find a better name.